src/components/LatestThreads/LatestThreads.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from "react";
+//import {RemoveScrollBar} from 'react-remove-scroll-bar';
+import {
+    createStyles,
+    makeStyles,
+    Typography,
+    Divider,
+    Box,
+    List,
+    ListItem,
+    ListItemText
+} from "@material-ui/core";
+import ForumIcon from "@material-ui/icons/Forum";
+
+const useStyles = makeStyles(() => createStyles({
+    root: {
+        marginTop: "30px",
```

Comment:
Remove that. Positioning of the component should be done in HomePage (encapsulate LatestThreads in Homepage with Box for example)